.github/workflows/ci.yaml

```diff
@@ -0,0 +1,57 @@
+name: ci
+
+on:
+  pull_request:
+
+jobs:
+  check-for-chart-changes:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Lint Bash scripts
+        uses: docker://koalaman/shellcheck-alpine:v0.7.0
+        with:
+          args: /github/workspace/.ci/lint-scripts.sh
+      - name: Check for chart changes
+        run: /home/runner/work/helm-charts/helm-charts/.ci/check-for-chart-changes.sh
+
+  lint-chart:
+    runs-on: ubuntu-latest
+    needs: check-for-chart-changes
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Run chart-testing (lint)
+        uses: helm/chart-testing-action@master
+        with:
+          command: lint
+          config: /home/runner/work/helm-charts/helm-charts/.ci/ct.yaml
+
+  install-chart:
+    name: install-chart
+    runs-on: ubuntu-latest
+    needs: lint-chart
+    strategy:
+      matrix:
+        k8s:
+          - v1.11.10
+          - v1.12.10
+          - v1.13.10
+          - v1.14.6
+          - v1.15.3
+          - v1.16.2
+          - v1.17.0
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Create kind cluster
+        uses: helm/kind-action@master
+        with:
+          config: /home/runner/work/helm-charts/helm-charts/.ci/kind-config.yaml
```

Comment:
`kind-config.yml` is no part of this PR, is it?

Reply:
Could you add the missing `kind-config.yaml` (and also add the license header to it?) ? :)

Reply:
also yaml files need the license header?

Reply:
kind-config.yaml was missing because of: https://github.com/eclipse/packages/blob/master/.gitignore#L10

Reply:
ah, damn .. thanks - will adjust the .gitignore